javav2/example_code/rds/Readme.md

```diff
@@ -2,19 +2,47 @@
+The credential provider used in all code examples is ProfileCredentialsProvider. For more information, see [Using credentials](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/credentials.html).
 
-To run these examples, you can setup your development environment to use Apache Maven or Gradle to configure and build AWS SDK for Java projects. For more information, 
+To run these examples, setup your development environment to use Apache Maven or Gradle to configure and build AWS SDK for Java projects. For more information, 
 see [Get started with the AWS SDK for Java 2.x](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/get-started.html).
 
+### Single action
+
+You will find these examples that use the **RdsClient** object: 
+
+- [Create an Amazon RDS instance and wait for it to be in an available state.](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/rds/src/main/java/com/example/rds/CreateDBInstance.java) (createDBInstance command)
+- [Create an Amazon RDS snapshot.](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/rds/src/main/java/com/example/rds/CreateDBSnapshot.java) (createDBSnapshot command)
+- [Delete an Amazon RDS instance.](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/rds/src/main/java/com/example/rds/DeleteDBInstance.java) (deleteDBInstance command)
+- [Retrieve attributes that belongs to an Amazon RDS account.](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/rds/src/main/java/com/example/rds/DescribeAccountAttributes.java) (describeAccountAttributes command)
+- [Describe an Amazon RDS instance.](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/rds/src/main/java/com/example/rds/DescribeDBInstances.java) (describeDBInstances command)
+- [Modify an Amazon RDS instance.](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/rds/src/main/java/com/example/rds/ModifyDBInstance.java) (modifyDBInstance command)
+- [Generate an authorization token for AWS Identity and Access Management (IAM) authentication to an Amazon RDS database.](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/rds/src/main/java/com/example/rds/GenerateRDSAuthToken.java) (generateAuthenticationToken command)
+- [Reboot an Amazon RDS instance.](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/rds/src/main/java/com/example/rds/RebootDBInstance.java) (rebootDBInstance command)
+- [Start an Amazon RDS instance.](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/rds/src/main/java/com/example/rds/startDatabase.java) (startDatabase command)
+- [Stop an Amazon RDS instance.](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/rds/src/main/java/com/example/rds/stopDatabase.java) (stopDatabase command)
```

Comment:
I think these links are usually in alphabetical order. Also, recommend deleting periods from these link names.